src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts

```diff
@@ -42,6 +43,7 @@ export class TableDesignerInput extends EditorInput {
 				this._onDidChangeDirty.fire();
 			}
 		}));
+		this._tableIconType = tableInfo.tableIconType;
```

Comment:
please handle the default scenario

Reply:
The default value is handled in the extension entry point where it defaults to `Basic`. I tried defaulting here but it gives a module error for azdata

Reply:
you have the default value logic in mssql extension, but it won't work for another extension. e.g. pgsql. the error itself is expected, you can't reference azdata here as it does not exist at runtime. you have to use the type you defined in sqlexthost, if that also doesn't work (if i remember correctly), you have to define the same enum again in this file.